### PR TITLE
Add timers for upcoming assets

### PR DESCRIPTION
**Pull request by Space-zee**
https://app.clickup.com/t/860q0f0xh

### CODE REVIEW

File: api/src/features/transactionsHistory/modules/assetHistory/assetHistory.service.ts [hunk 1]
```diff
@@ -267,7 +267,7 @@ export class AssetHistoryService {
       );
       this.logger.debug(`Saved NewAsset event. asset address:${logArgs.asset}`);
     } catch (e) {
-      this.logger.error(`NewAsset History Regression Error: ${e}`);
+      this.logger.debug(`NewAsset History Regression Error: ${e}`);
```
Comment (dmitriy-green) [inline api/src/features/transactionsHistory/modules/assetHistory/assetHistory.service.ts:270]: is this intentional?

Comment (Space-zee) [reply api/src/features/transactionsHistory/modules/assetHistory/assetHistory.service.ts:270]: yes

**Review by boivlad — DISMISSED** [review]

**Review by dmitriy-green — DISMISSED** [review]

File: frontend/app/features/admin/views/EditAsset/EditAssetSaleDate/index.tsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+import {Button} from "../../../../../shared/ui/views/Button";
+import clsx from "clsx";
+import {ArrowIconWithoutLine} from "../../../../../shared/ui/views/ArrowIconWithoutLine";
+import React from "react";
+import {DatePickerWithTime} from "../../../../../shared/ui/views/DatePickerWithTime";
+
+type Props = {
+  onChange: (inputName: string, inputValue: string) => void;
+  value: Date | null,
+  continueButton: () => void,
```
Comment (ruslanpalagin) [inline frontend/app/features/admin/views/EditAsset/EditAssetSaleDate/index.tsx:12]: IMHO 
onContinueClick is much better name.

File: frontend/app/features/admin/controllers/AdminEditAsset/index.tsx [hunk 3]
```diff
@@ -157,6 +160,50 @@ export const AdminEditAssetController = () => {
...
+        llcAdministrationFeePercentage: editForm.values.llcAdministrationFee,
+        saleDate: editForm.values.saleDate
+      }
+      const response = await httpClient.fetch(apiPaths.updateAsset(), {
+        body: {
+          attrs: data
+        },
+        method: 'PATCH'
+      })
+      console.log('response', response)
```
Comment (ruslanpalagin) [inline frontend/app/features/admin/controllers/AdminEditAsset/index.tsx:201]: rm log

File: frontend/app/features/admin/controllers/AdminEditAsset/index.tsx [hunk 4]
```diff
@@ -35,12 +36,13 @@ export const AdminEditAssetController = () => {
   const [sendDataStatus, setSendDataStatus] = useState<number>()
   const [activeTab, setActiveTab] = useState("generalInfo");
   const [assetStatus, setAssetStatus] = useState<string>('')
-  const [assetPrice, setAssetPrice] = useState<string>('')
+  const [assetPrice, setAssetPrice] = useState<string>('');
+  const [updateSaleDateStatus, setUpdateSaleDateStatus] = useState(false)
```
Comment (ruslanpalagin) [inline frontend/app/features/admin/controllers/AdminEditAsset/index.tsx:40]: Each form typed by UiForm generic has "touches" which should be used to track changed fields if we need to.
Please use touches in a future where it makes sense, thanks

File: frontend/app/features/admin/views/EditAsset/EditAssetSaleDate/index.tsx [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+  sendDataStatus: boolean,
+  sendAllData: () => void
```
Comment (ruslanpalagin) [inline frontend/app/features/admin/views/EditAsset/EditAssetSaleDate/index.tsx:14]: sendAllData -> onSendAllDataClick

File: frontend/app/features/admin/views/EditAsset/EditAssetSaleDate/index.tsx [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+import clsx from "clsx";
+import {ArrowIconWithoutLine} from "../../../../../shared/ui/views/ArrowIconWithoutLine";
+import React from "react";
+import {DatePickerWithTime} from "../../../../../shared/ui/views/DatePickerWithTime";
+
+type Props = {
+  onChange: (inputName: string, inputValue: string) => void;
+  value: Date | null,
+  continueButton: () => void,
+  sendDataStatus: boolean,
```
Comment (ruslanpalagin) [inline frontend/app/features/admin/views/EditAsset/EditAssetSaleDate/index.tsx:13]: sendDataStatus -> isSellDateChanged
OR even better
sendDataStatus -> showSellDateInput

**Review by ruslanpalagin — APPROVED** [review]

File: frontend/app/features/admin/views/EditAsset/EditAssetSaleDate/index.tsx (hunk 2 above)
Comment (Space-zee) [reply frontend/app/features/admin/views/EditAsset/EditAssetSaleDate/index.tsx:12]: +

File: frontend/app/features/admin/controllers/AdminEditAsset/index.tsx (hunk 3 above)
Comment (Space-zee) [reply frontend/app/features/admin/controllers/AdminEditAsset/index.tsx:201]: +

File: frontend/app/features/admin/views/EditAsset/EditAssetSaleDate/index.tsx (hunk 5 above)
Comment (Space-zee) [reply frontend/app/features/admin/views/EditAsset/EditAssetSaleDate/index.tsx:14]: +

File: frontend/app/features/admin/views/EditAsset/EditAssetSaleDate/index.tsx (hunk 6 above)
Comment (Space-zee) [reply frontend/app/features/admin/views/EditAsset/EditAssetSaleDate/index.tsx:13]: +

File: frontend/app/features/admin/controllers/AdminEditAsset/index.tsx (hunk 4 above)
Comment (Space-zee) [reply frontend/app/features/admin/controllers/AdminEditAsset/index.tsx:40]: need to fix in future

**Review by ruslanpalagin — APPROVED** [review]

**Pull request merged**